### DISCUSSION TITLE
How to get something custom to show on my home screen left side bar

### DISCUSSION THREAD

**Discussion by ghost** [squidfunk/mkdocs-material#7388] [Q&A]
<img width="1440" alt="Screenshot 2024-07-22 at 6 51 19 PM" src="https://github.com/user-attachments/assets/2a104fdb-8711-4098-8bae-4bb43e511117">
as you can see in this mkdocs site, on the home page there is a custom link thing on the left side and i was wondering how i can acheive that for my own site.

**Comment by alexvoss** [comment]
Not sure which bit is the custom link but In the `nav` element of your `mkdocs.yml` you can add external links as well as links to your own pages. Have a look in the [MkDocs documentation on `nav`](https://www.mkdocs.org/user-guide/configuration/#nav)

**Reply by ghost** [reply]
the one on the left side stating
"pack guides
all the mods 9
ect..."
how do i get something like that? also thanks for replying ill go check out the link

**Reply by alexvoss** [reply]
That is the result of choosing the relevant options for the main navigation. We have a [setup guide](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/) for that - and defining a navigation structure in the `mkdocs.yml`. Note that the titles used in the navigation can come from that navigation definition but can also be defined in the pages (either via a title metadata element or via the first heading).

**Reply by ghost** [reply]
Alright thanks a lot
